RoaringBitmap/src/main/java/org/roaringbitmap/RoaringBitmap.java

```diff
@@ -2845,9 +2845,11 @@ public long previousValue(int fromValue) {
   @Override
   public long nextAbsentValue(int fromValue) {
     long nextAbsentBit = computeNextAbsentValue(fromValue);
-    assert nextAbsentBit <= 0xFFFFFFFFL;
-    assert nextAbsentBit >= Util.toUnsignedLong(fromValue);
-    assert !contains((int) nextAbsentBit);
+    if(nextAbsentBit == 4294967296L) {
```

Comment:
0x100000000 or 1L << 32 would be easier to read here.

Reply:
Right. And I don't think I need the contains....

Reply:
@richardstartin Please check the next commit:
https://github.com/RoaringBitmap/RoaringBitmap/pull/720/commits/a1c91153064629c1424b562a75c7d312b9cc7b1a